interfaces/cython/cantera/test/test_reaction.py

```diff
@@ -4,6 +4,58 @@
 from . import utilities
 
 
+class TestImplicitThirdBody(utilities.CanteraTest):
+
+    @classmethod
+    def setUpClass(cls):
+        utilities.CanteraTest.setUpClass()
+        cls.gas = ct.Solution('gri30.yaml')
+
+    def test_implicit_three_body(self):
+        yaml1 = """
+            equation: H + 2 O2 <=> HO2 + O2
+            rate-constant: {A: 2.08e+19, b: -1.24, Ea: 0.0}
+            """
+        rxn1 = ct.Reaction.fromYaml(yaml1, self.gas)
+        self.assertEqual(rxn1.reaction_type, 'three-body')
```

Comment:
Shouldn't this also test that the default efficiency is zero and that efficiencies is equal to `{"O2": 1}`?

Reply:
it does this implicitly with the checks right below, but it won't hurt to add it explicitly.